docs/source/library.rst

```diff
@@ -13,9 +13,11 @@ gammasim-tools is composed of the following modules:
   :maxdepth: 1
   :glob:
 
-  simulators
-  mc_model
-  ray_tracing
   camera_efficiency
+  configuration
+  data_model
   layout_array
+  mc_model
+  ray_tracing
+  simulators
   util
```

Comment:
I am wondering about the modules in general. I would expect to have in this file all the listed modules, or at least all the repositories with modules. I propose we try to adjust this list to the code and dont come up with new names and categories that are not in the code. For instance, in the code we have `simulator.py` and here `simulators`. I also haven't seen the db_handler here. Should we first think about reordering the modules in the code and adjust the list here accordingly?